style.css

```diff
@@ -0,0 +1,285 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&family=Roboto:wght@100;300;400;500&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  display: flex;
+  width: 375px;
+  height: 48px;
```

Comment:
[Optional] Kindly consider using percentages or em instead of pixels as much as possible. The reason for this preference is that using relative units like percentages or rem provides better scalability and responsiveness across different devices and screen sizes 😁 .